tests/conftest.py

```diff
@@ -1,13 +1,125 @@
 from pathlib import Path
 from typing import Iterator
 
+import numpy as np
+import pandas as pd
 import pytest
 
-from tests import acceptance_test_data
+from ra2ce.analysis.analysis_config_data.enums.risk_calculation_mode_enum import (
+    RiskCalculationModeEnum,
+)
+from ra2ce.analysis.losses.risk_calculation.risk_calculation_cut_from_year import (
+    RiskCalculationCutFromYear,
+)
+from ra2ce.analysis.losses.risk_calculation.risk_calculation_default import (
+    RiskCalculationDefault,
+)
+from ra2ce.analysis.losses.risk_calculation.risk_calculation_triangle_to_null_year import (
+    RiskCalculationTriangleToNullYear,
+)
+from tests import acceptance_test_data, test_data
 
 
 @pytest.fixture(name="valid_analysis_ini")
 def _get_valid_analysis_ini_fixture() -> Iterator[Path]:
     _ini_file = acceptance_test_data.joinpath("analyses.ini")
     assert _ini_file.exists()
     yield _ini_file
+
+
+@pytest.fixture(name="losses_fixture")
+def get_losses_fixture() -> Iterator[pd.DataFrame]:
+    _csv_results_filepath = test_data.joinpath(
+        "losses", "csv_data_for_losses", "results_test_calc_vlh.csv"
+    )
+    assert _csv_results_filepath.exists()
+    _test_calc_vlh = pd.read_csv(_csv_results_filepath)
+    yield _test_calc_vlh.drop(
+        columns=[
+            "EV1_ma",
+            "EV2_mi",
+            "vlh_business_EV1_ma",
+            "vlh_commute_EV1_ma",
+            "vlh_business_EV2_mi",
+            "vlh_commute_EV2_mi",
+            "risk_vlh_total_default",
+            "risk_vlh_total_triangle_to_null_year",
+            "risk_vlh_total_cut_from_year",
+        ]
+    )
+
+
+@pytest.fixture(name="expected_factory_results")
```

Comment:
Please add fixtures on the lowest possible level based on where they are used. In this case tests/analysis/losses/risk_calculation/conftest.py.
Also holds for other fixtures in this module.